src/Support/Http/Interface/ResponseHandlerInterface.php

```
@@ -0,0 +1,34 @@
+     * exception based on the HTTP status code.
+     *
+     * @param  ResponseInterface  $response  The raw PSR-7 response from the HTTP client.
+     *                                       * @return array<string, mixed> The decoded JSON body.
+     *
+     * @throws \AsaasPhpSdk\Exceptions\Api\AuthenticationException When status is 401.
+     * @throws \AsaasPhpSdk\Exceptions\Api\ValidationException When status is 400.
+     * @throws \AsaasPhpSdk\Exceptions\Api\NotFoundException When status is 404.
+     * @throws \AsaasPhpSdk\Exceptions\Api\RateLimitException When status is 429.
+     * @throws \AsaasPhpSdk\Exceptions\Api\ApiException For 5xx errors or unexpected status codes.
+     */
+    public function handle(ResponseInterface $response): array;
```

PHPStan (2.1.37)

[error] Method AsaasPhpSdk\Support\Http\Interface\ResponseHandlerInterface::handle() return type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)